drivers/util.py

```diff
@@ -78,11 +78,18 @@ class SMException(Exception):
 
 
 class CommandException(SMException):
+    def error_message(self, code):
+        if code > 0:
+            return os.strerror(code)
+        elif code < 0:
+            return "Signalled %s" % (abs(code))
```

Comment:
It would be helpful to use `signal.strsignal()` if we could use Python 3.8.

Reply:
that it would, but a lot more work required to get this to be able to run in Python 3.8